src/GT.UI/Controllers/ListingsController.cs

```diff
@@ -10,50 +12,59 @@ namespace GT.UI.Controllers
 	[ApiController]
 	public class ListingsController : ControllerBase
 	{
-		private readonly GTAppContext _context;
+		private readonly IGTListingService _ListingService;
+		private readonly UserManager<IdentityUser> _userManager;
 
-		public ListingsController(GTAppContext context)
-		{
-			_context = context;
+
+		public ListingsController(IGTListingService iGTListingService, UserManager<IdentityUser> userManager)
+{
+			_ListingService = iGTListingService;
+			_userManager = userManager;
 		}
 
 		// GET: api/Listings
 		[HttpGet]
 		public async Task<ActionResult<IEnumerable<Listing>>> GetListings()
```

Comment:
Should return a IEnumerable<ListingPartialDTO> and pass in a IListingFilterModel for listing filtering. Filter needs to get passed down to GetAsync.

Reply:
Its returned wraped in the status code. If we were to return only IEnumerable we would have to throw an exception incase the list is null instead of a NotFound code. I belive this is better for error handling but perhaps this should be discussed

Reply:
ActionResult<IEnumerable<ListingDTO>>